features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayConfiguration.kt

```diff
@@ -149,6 +151,15 @@ data class SessionReplayConfiguration internal constructor(
             return this
         }
 
+        /**
+         * This option controls whether optimization is enabled or disabled for recording Session Replay data.
+         * By default the value is false, meaning the dynamic optimization is enabled.
+         */
+        fun disableDynamicOptimization(disableDynamicOptimization: Boolean): Builder {
```

Comment:
> [!WARNING]
> Having a "negative" name with a boolean can become confusing for users. When I pass in false, do I want the feature to be disable (meaning off) or not disabled. Also if the method is called `disabledX`, one could try to find a matching `enableX`. A better way would be to have something like `setDynamicOptimizationEnabled(enabled: Boolean)` which is easier to understand.

Reply:
After a discussion with @jonathanmos , we agree that to change the variable name to `dynamicOptimizationEnabled`

Reply:
I think it should be `setDynamicOptimizationEnabled` then, because we should be consistent with the lingvo used in the our API already. You can search for `*enable*` in `apiSurface` files to see how it should be consistent.

Reply:
good point, updated